components/director/pkg/jsonschema/validator.go

```diff
@@ -64,11 +65,10 @@ func (v *validator) ValidateRaw(value interface{}) (bool, error) {
 		return true, nil
 	}
 
-	jsonLoader := gojsonschema.NewRawLoader(value)
-	result, err := v.schema.Validate(jsonLoader)
+	valueString, err := json.Marshal(value)
 	if err != nil {
-		return false, errors.Wrapf(err, "while validating value %+v against schema %+v", value, v.schema)
+		return false, err
```

Comment:
IMO the error wrapper from the origin should be moved to the `ValidateString` func. Here the wrapping func should also be present.

Reply:
done